CONTRIBUTING.md

```diff
@@ -171,6 +171,41 @@ One other detail: MCprep uses git lfs or Large File Storage, to avoid saving bin
 
 Run into other gotchas? Please open a [new issue](https://github.com/TheDuckCow/MCprep/issues)!
 
+
+## Commit Messages
+Git commits should explain why a change was made, because the diff will show the changes made. For example, instead of writing:
+```
+Added ability to "import" MTL files
+```
+
+Instead do:
+```
+Added the ability to "import" MTL files
+
+MCprep's file explorer shows both OBJs and MTLs, and sometimes users end up clicking
+MTL files. This brings a quality of life improvement to change the extension
+if the file selected is an MTL, since MTLs share the same name as their corresponding
+OBJ files
+```
+
+The first line is a summary of the changes, and should be less then 50 characters. The rest should justify the changes. Convince us why these changes are important and why they've been made this way.
+
+Git won't automatically wrap messages either, so each line should have a limit of 72 characters.
+
+Here's a template I found that can help (modified for simplicity) by using # to define which is the limit Git can display for each line:
```

Comment:
```suggestion
Here's a template some MCprep developers found that can help (modified for simplicity) by using # to define which is the limit Git can display for each line:
```

Suggesting this to keep the guide feel neutral, since it is indeed the result of multiple of our contributions.